beater/cloudbeat.go

```diff
@@ -33,12 +35,13 @@ import (
 	csppolicies "github.com/elastic/csp-security-policies/bundle"
 	agentconfig "github.com/elastic/elastic-agent-libs/config"
 	"github.com/elastic/elastic-agent-libs/logp"
-
-	"github.com/gofrs/uuid"
 )
 
 const (
 	reconfigureWaitTimeout = 10 * time.Minute
+	flushInterval          = 10 * time.Second
+	eventsThreshold        = 75
+	resourceChBuffer       = 50
```

Comment:
```suggestion
	resourceChBuffer       = 10000
```
We don't want the writers to be blocked.